test/loaders.js

```diff
@@ -86,6 +87,11 @@ test('loaders create batching resolvers', async (t) => {
             name: 'Buddy'
           },
           params: {}
+        }, {
+          obj: {
+            name: 'Max'
+          },
+          params: {}
```

Comment:
This should not happen and it was what I feared about this implementation. By making `info` part of of the object, we are passing it through https://github.com/mcollina/single-user-cache/blob/master/index.js#L66 and the fetching of this entity is now duplicated.

As you can see, it has now the same output when the cache is disabled. __I think we should add the `info` object only if the cache is disabled__.